ui/src/views/Quickstart/index.jsx

```diff
@@ -47,11 +46,11 @@ const initialYaml = {
       metadata: {
         name: '',
         description: '',
-        owner: '{{ event.head.user.email }}',
-        source: '{{ event.head.repo.url }}',
+        owner: `{{ event.sender.login }}@users.noreply.github.com`,
```

Comment:
Please replace `{{ }}`  template with json-e

Reply:
I did all the changes.
This part is giving some error. I changed it to 
`owner: { $eval: 'event.sender.login' }}`@users.noreply.github.com`,`
What will be the json-e format for eval with string contact?

Reply:
I am noticing you switched from string to template literal (I am talking about JavaScript types)? Can you please switch back? :)

What exactly is the error?

I don't see json-e syntax here. I still see the `{{ }}` templates. Please let me know if you have any questions

Reply:
Do you mean change it like this
` owner: `${event.sender.login}@users.noreply.github.com`,`?

Reply:
Yes! Except please don't use template literal - there is no need for it. Use a string literal:

```
owner: '${event.sender.login}@users.noreply.github.com',
```

The part `${event.sender.login}` is not part of JavaScript's template literal, it's json-e language https://github.com/taskcluster/json-e . In the YML it needs to be a string.

Reply:
Doing like this
`owner: '${event.sender.login}@users.noreply.github.com'`
yarn lint is throwing this error
` Unexpected template string expression (no-template-curly-in-string)`

Reply:
Disable that linter rule for the affected code